extras/rya.export/README.md

```diff
@@ -0,0 +1,250 @@
+
+The MongoDB side export uses a DB object to search the collection for a Rya statement while sorting them by timestamp.
+
+By default the tool will copy all the tables from the parent.
+
+When merging, the Rya Export Client Accumulo statement store utilizes its internal DAO to handle the processing of data. It is implemented using Java with a Scanner running over the child Accumulo instance to find the statements to copy and then queries for the same statements in the parent instance. If the statement key is found then it doesn't need to merge the statement and it proceeds to the next statement. The column visibility is combined with an OR operation if only the visibilities are different between the child and parent.
+
+If a key is found in the parent that doesn't exist in the child instance then we need to determine whether it was added by the parent or deleted by the child. If the timestamp is before the start time then the child deleted the key and the parent should delete the key as well. If the timestamp is after the start time then the parent added it and the key should remain. Similar time comparisons are applied too if a key exists in the child instance but not the parent.
+
+### Merge Logic
+
+**Parent:** Original Rya repository. Ignore modifications.  
+**Child:** Rya repository that was copied from parent then modified.  
+**older, newer:** Statement timestamp is older/newer than specified merge time (start time).  
+**missing:** Statement is absent from the repository.  
+
+| **case** | **Parent** | **Child** | **assume that** | **merge modification** | **in parent after** |
+|----------|------------|-----------|-----------------|------------------------|---------------------|
+| 1        | older      | missing   | child deleted   | delete from parent     | no                  |
+| 2        | newer      | missing   | parent added    | do nothing             | yes                 |
+| 3        | missing    | older     | parent deleted  | do nothing             | no                  |
+| 4        | missing    | newer     | child added     | add to parent          | yes                 |
+| 5x       | older      | older     | same key        | do nothing             | yes                 |
+| 6x       | newer      | newer     | same key        | do nothing             | yes                 |
+| 7\*      | older      | newer     | child updated   | do cases 1 and 4       | yes                 |
+| 8\*      | newer      | older     | parent updated  | do cases 2 and 3       | yes                 |
+
+NOTE:
+
+**x** - The two cases having the same key and timestamp are already merged.
+
+**\*** - The last two cases are really two distinct keys, since the timestamp is part of the Accumulo key. They are handled as two comparisons. They are different versions of the same key, but Rya is generally used with only the latest version.
+
+Here it is again, in full English:
```

Comment:
I would have this above the table, title to table with 'To Simplify' and get rid of the 'Here it si again, in full English'